tests/Resources/ODS.robot

```diff
@@ -400,3 +400,30 @@ Delete RHODS Config Map
     [Arguments]     ${name}  ${namespace}=redhat-ods-applications
     OpenShiftLibrary.Oc Delete    kind=ConfigMap  name=${name}  namespace=${namespace}
 
+Wait Until Cuda Builds Are Completed
+    [Documentation]    Waits until pytorch and tensorflow builds have Complete Status.
+    ...    The keyword waits 20 minutes for each build to be in that status, failing otherwise.
+
+    ${pytorch_build_name}=    Search Last Build    namespace=redhat-ods-applications
+    ...    build_name_includes=pytorch
+    Wait Until Build Status Is    namespace=redhat-ods-applications
+    ...    build_name=${pytorch_build_name}    expected_status=Complete
+
+    ${tensorflow_build_name}=    Search Last Build    namespace=redhat-ods-applications
+        ...    build_name_includes=tensorflow
+    Wait Until Build Status Is    namespace=redhat-ods-applications
+    ...    build_name=${tensorflow_build_name}    expected_status=Complete
+
+Verify Cuda Builds Are Completed
+    [Documentation]    Verify All Cuda Builds have status as Complete
+    ${Pods} =    Run    oc get build -n redhat-ods-applications
```

Comment:
Could you change the run oc for the openshift-library  `Oc get  kind=Build   namespace=redhat-ods-applications`

Reply:
I've deleted this keyword and instead I've added "Verify All Builds Are Complete" in Builds.robot. This one is using the openshift-library

Reply:
In the end I've removed the call to "Verify All Builds Are Complete" because it was not needed after "Wait Until All Builds Are Complete"